src/main/scala/cognite/spark/v1/SequencesRelation.scala

```diff
@@ -48,9 +48,27 @@ class SequencesRelation(config: RelationConfig)(val sqlContext: SQLContext)
         s.dataSetId
       )
     }
-    client.sequences
-      .create(sequences)
-      .flatTap(_ => incMetrics(itemsCreated, sequences.size)) *> IO.unit
+    val nbColumns = sequences.flatMap(_.columns.toList).size
+    if (nbColumns < Constants.DefaultSequencesTotalColumnsLimit) {
+      client.sequences
+        .create(sequences)
+        .flatTap(_ => incMetrics(itemsCreated, sequences.size)) *> IO.unit
+    } else {
+      // Sequence API does not support more than 200 columns per sequence,
+      // and the maximum total columns per request is 10000
+      // so grouped by 50 ensure that each chunk can not have more than 10000 columns in total,
+      // otherwise we already reject because of the first condition
```

Comment:
Hm, I don't see where we do that rejection.

Reply:
The limit of 200 columns `[is specified in the API ](https://docs.cognite.com/api/v1/#operation/createSequence)`
But the limit about 10000 columns max per request is not, yesterday I talked with time-lord team and they just create a ticket for this https://cognitedata.atlassian.net/browse/CDF-14714 :)